docs/client/_client-sdk-template.mdx

```diff
@@ -1,8 +1,8 @@
-import Tabs from '@theme/Tabs';
-import TabItem from '@theme/TabItem';
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
```

Comment:
we gotta get on the same prettier otherwise everyones diff will look like this.

Can you revert all of these?  Seem unnecessary